### PR TITLE
Add Todo model module

### PR DESCRIPTION
**Pull request by jeffreyjurgajtis**

### CODE REVIEW

File: test/models/todo_test.exs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+defmodule TodoApi.TodoTest do
+  use TodoApi.ModelCase
+
+  alias TodoApi.Todo
+  alias TodoApi.Repo
+
+  test "changeset valid? returns true when content and user_id are present " do
+    attrs = %{completed_at: "2010-04-17 14:00:00", content: "hey", user_id: 1}
```
Comment (cmoel) [inline test/models/todo_test.exs]: Do we need `completed_at` to be set for this to pass?

**Comment by cmoel** [comment]
In the tests, can we use the `Ecto.DateTime` struct for `completed_at` values in tests (since I believe that's what it's going to happen internally)? I believe you should be able to use `Ecto.DateTime.local` or `Ecto.DateTime.utc`

**Comment by jeffreyjurgajtis** [comment]
Yep, passing `Ecto.DateTime` structs for `completed_at` values does work. Tests have been updated.

**Comment by cmoel** [comment]
:+1:

**Pull request merged**